spec/views/alaveteli_pro/dashboard/_projects.html.erb_spec.rb

```diff
@@ -0,0 +1,107 @@
+# -*- encoding : utf-8 -*-
+require 'spec_helper'
+
+describe "alaveteli_pro/info_requests/dashboard/_projects.html.erb" do
+  let(:pro_user) { FactoryGirl.create(:pro_user) }
+
+  before do
+    FactoryGirl.create(:info_request, user: pro_user)
+    FactoryGirl.create(:waiting_clarification_info_request, user: pro_user)
+    FactoryGirl.create(:successful_request, user: pro_user)
+    FactoryGirl.create(:error_message_request, user: pro_user)
+    FactoryGirl.create(:awaiting_description, user: pro_user)
+    FactoryGirl.create(:overdue_request, user: pro_user)
+    FactoryGirl.create(:very_overdue_request, user: pro_user)
+
+    FactoryGirl.create(:draft_info_request, user: pro_user)
+
+    public_bodies = FactoryGirl.create_list(:public_body, 10)
+    FactoryGirl.create(:info_request_batch, user: pro_user,
+                                            public_bodies: public_bodies)
+    FactoryGirl.create(:draft_info_request_batch, user: pro_user,
+                                                  public_bodies: public_bodies)
+
+    expect(pro_user.request_summaries.category(:response_received).count).to eq 1
```

Comment:
Whoops - a bit of debugging I forgot to remove. Fixed in b0db9aec123086c40cac311cc1cfcb200e9ad2f6